public/static/locales/el/common.json

```
@@ -0,0 +1,19 @@
+{
+	"slogan": "H πρώτη Ελληνική λέσχη πληροφορικής για εφήβους",
+	"titlePrefix": "Είμαστε η",
+	"articleQuestion": "Σε ενδιαφέρει η πληροφορική;",
+	"description": "Η Hackropolis είναι μία διαδικτυακή λέσχη πληροφορικής για ελληνόφωνους εφήβους που ενδιαφέρονται για την πληροφορική και άλλους σχετικούς τομείς. Οργανώνουμε συχνά διαδικτυακές συναντήσεις και φτιάχνουμε μαζί εφαρμογές και προγράμματα. Δεν απαιτείται προυπάρχουσα εμπειρία - το μονο που χρειάζεσαι είναι σύνδεση στο διαδίκτυο!",
+	"operateUnder": "Αποτελούμαστε κυρίως από μαθητές δευτεροβάθμιας και τριτοβάθμιας εκπαίδευσης και λειτουργούμε υπό την αιγίδα του",
+	"nonProfit": "(μίας μη κερδοσκοπική οργάνωση στην Καλιφόρνια, ΗΠΑ), που υποστηρίζει λέσχες σαν την δική μας ανά τον κόσμο",
```

Should it be
`(μίας μη κερδοσκοπική οργάνωση στην Καλιφόρνια, ΗΠΑ)` or `(μία μη κερδοσκοπική οργάνωση στην Καλιφόρνια, ΗΠΑ)`?

If the first, it's missing some "ς"s:
```suggestion
	"nonProfit": "(μίας μη κερδοσκοπικής οργάνωσης στην Καλιφόρνια, ΗΠΑ), που υποστηρίζει λέσχες σαν την δική μας ανά τον κόσμο",
```